internal/export/db.go

```diff
@@ -20,8 +20,6 @@ import (
 type DBClient interface {
 	CloseSession()
```

Comment:
If creation is a detail outside the scope of the contract, it feels like destruction should be as well.

Reply:
Destruction is dependent on the instance though. Creation happens before the instance exists. Very metaphysical stuff ;-)

Reply:
Just as creation is.  I guess what I'm saying is we should be instantiating and destructing at the same level.  Since we create using a switch, perhaps we should destruct at that same level using a switch.